### PR TITLE
Client side icon support for vue.

### PR DESCRIPTION
**Pull request by SahilTara**
Summary of changes:
Implements icon support for vue and its pages.

To test PR will need to build.

Should be merged/reviewed after #617 is in.

Related issue:
#199

### CODE REVIEW

**Review by Tanya0609 — COMMENTED** [review]
The Vue icon shows in clinet side. Code looks good but some portion overlaps with another PR #631.
Will approve this PR once the other PR is merged and this one gets rebased 👍

**Review by streetcoder123 — APPROVED** [review]
This looks Good. Works as expected!

**Review by Tanya0609 — APPROVED** [review]

**Review by annielovesjs — APPROVED** [review]

**Pull request merged**
